tardis/model/tests/data/csvy_model_to_test_abundances.csvy

```diff
@@ -23,9 +20,13 @@ datatype:
        desc: fractional Ni56 abundance
 
 v_inner_boundary: 9000 km/s
-v_outer_boundary: 12000 km/s
+v_outer_boundary: 16000 km/s
 ---
-velocity,density,t_rad,H,He,Ni56
-9000, 5e-10, 7000, 1.0, 1.0, 1.0
-10500, 2.0e-10, 7000, 0.0, 0.99, 0.01
-12000, 9e-11, 7000, 0.4, 0.58, 0.02
+velocity,density,H,He,Ni56
+9000,  5e-10,   0.5,  0.5,  0
+10500, 2.0e-10, 0.0,  0.98, 0.02
+12000, 9e-11,   0.33, 0.64, 0.03 
+13000, 9e-12,   0.3,  0.6,  0.1
+14000, 6e-12,   0.5,  0.4,  0.1
+15000, 4e-12,   0.4,  0.55, 0.05
+16000, 3e-12,   0.2,  0.79, 0.01
```

Comment:
I guess this is why you changed the v_outer_boundary.  I'm not sure why we need this larger ejecta model for these tests though.